src/lib/functions.js

```diff
@@ -0,0 +1,57 @@
+//(x,y) will be coordinates of a square
+export const free = (board, x,y) => {
+  return board[x][y] === '0'
+} // check if a square is free
+
+
+export const rows = (board) => {
```

Comment:
why do we have to define the board?

Reply:
Actually, it's just taking the rows from the board which is in fact the board itself :smiley_cat: